playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -151,22 +151,19 @@
   tags:
     - run
 
-- name: Create .config/openstack directory
+- name: Ensure /etc/openstack directory exists
+  become: yes
   file:
-    path: /root/.config/openstack
+    path: "{{ item }}"
     state: directory
-    owner: root
-    group: root
-    mode: '0755'
-  tags:
-    - run
+  with_items:
+    - /etc/openstack
```

Comment:
only one item? why not use the previous path: directly?

Reply:
Could be. I just copied the code from airship task